modules/server/src/main/scala/navigate/server/tcs/WfsEpicsSystem.scala

```diff
@@ -47,15 +48,26 @@ object WfsEpicsSystem {
         new WfsEpicsImpl[F](channels),
         timeout
       )
+
+    override def getQualityStatus: WfsQualityStatus[F] = new WfsQualityStatus[F] {
+      override val flux: VerifiedEpics[F, F, Int]                 =
+        VerifiedEpics.readChannel(channels.telltale, channels.flux)
+      override val centroidDetected: VerifiedEpics[F, F, Boolean] = VerifiedEpics
```

Comment:
Are centroids calculated directly on the TCS?

Reply:
No, both values come from the WFS. Which is why they are in `WfsEpicsSystem` and not `TcsEpicsSystem`.